summoner-cli/src/Summoner/Template/Stack.hs

```diff
@@ -36,5 +36,4 @@ stackFiles Settings{..} = map createStackYaml settingsTestedVersions
             $endLine
 
             ghc-options:
-              "$$locals": -fhide-source-paths
```

Comment:
The previous two lines are not needed anymore, as there are no other `ghc-options`.